src/controller/CHIPDevice.h

```diff
@@ -178,7 +183,7 @@ class DLL_EXPORT Device
     void Init(DeviceTransportMgr * transportMgr, SecureSessionMgr * sessionMgr, Inet::InetLayer * inetLayer, uint16_t listenPort,
               NodeId deviceId, const Transport::PeerAddress & peerAddress, Transport::AdminId admin)
     {
-        Init(transportMgr, sessionMgr, inetLayer, mListenPort, admin);
+        Init(transportMgr, sessionMgr, nullptr, inetLayer, mListenPort, admin);
```

Comment:
I don't think that makes sense.  Why can we get away with not having an exchange manager here?

Reply:
This Init is used by RendezvousSession, which still run on SecureSessionMgr directly. ExchangeMgr is not used by SecureSessionMgr. We will replace legacy pairing with clustered based network provisioning, this Init function will be deleted together with RendezvousSession. Right now, just init ExchangeMgr here to keep consistency.